bin/bootstrax

```diff
@@ -538,11 +544,19 @@ def infer_target(rd):
 
     n_fails = rd['bootstrax'].get('n_failures', 0)
     if n_fails:
-        log.debug(f'override_target::\tdeleting targets')
+        log.debug(f'Deleting targets')
         for _rm_t, _rm_t_after in remove_target_after_fails.items():
-            _t = [__t for __t in _t if (_rm_t not in __t and n_fails < _rm_t_after)]
-            _pp = [__pp for __pp in _pp if (_rm_t not in __pp and n_fails < _rm_t_after)]
+            for __i, __t in enumerate(_t):
+                if __t in _rm_t and n_fails > _rm_t_after:
```

Comment:
Is the logic here the right way around? Previously this was `_rm_t in __t`

Reply:
Logic is reverted, now this `if` -> deletes targets if we want to, whereas it used to say `if` -> keep the target. Therefore there is also no `not`  involved for `__t in _rm_t`